script.py

```diff
@@ -1,37 +1,62 @@
 import datetime
 import hashlib
+import requests
 
 
-def totimestamp(dt, epoch=datetime.datetime(1970,1,1)):
+def totimestamp(dt, epoch=datetime.datetime(1970, 1, 1)):
     td = dt - epoch
     # return td.total_seconds()
-    return (td.microseconds + (td.seconds + td.days * 86400) * 10**6) / 10**6 
+    return (td.microseconds + (td.seconds + td.days * 86400) * 10**6) / 10**6
```

Comment:
You can also convert `86400` to a humanly understandable number `DAY = 60 * 60 * 24`